configure.ac

```diff
@@ -556,6 +556,11 @@ if test x"$GCC" = xyes; then
 		# The runtime code does not respect ANSI C strict aliasing rules
 		CFLAGS="$CFLAGS -fno-strict-aliasing"
 
+		# Compliant C++ unwinding code will stop at any frame with CANTUNWIND in the
+		# FDE. If we don't include -fexceptions, some of the runtime functions end up 
+		# having the CANTUNWIND attribute. This is necessary for the C++ EH backend.
+		CFLAGS="$CFLAGS -fexceptions"
```

Comment:
This will definitely increase code size.
Also, why doesn't this break on x86/amd64 linux ?